site/pages/review-disruption/[disruptionId].page.tsx

```diff
@@ -408,6 +409,15 @@ const ReviewDisruption = ({
                         >
                             Delete disruption
                         </button>
+                        <button
```

Comment:
This could just be a link back to the dashboard page since we don't need to do anything in the API here

Reply:
done